sources/api/schnauzer/src/helpers.rs

```diff
@@ -132,6 +133,21 @@ mod error {
             template: String,
             source: std::io::Error,
         },
+
+        #[snafu(display(
+            "Expected an absolute URL, got '{}' in template '{}': '{}'",
```

Comment:
The proxy variables typically allow just a hostname, and don't require a full URL, no?  I'm wondering if we can just use url::Host to validate.  It accepts IPs.  (We use it for DNSDomain, and explicitly reject IPs.)

Reply:
My intention for this helper was to trim `settings.kubernetes.api-server` which should always be a fully qualified domain name (otherwise kubelet complains).

I'll go ahead and make it more flexible.

Reply:
After some tinkering and thinking, I think I'm going to keep it as is. This helper is really just meant for trimming the URI scheme and paths off of absolute URLs stored in settings (e.g. kubernetes.api-server) so they can be included in the `no-proxy` setting.